lib/private/AppFramework/Middleware/Security/SecurityMiddleware.php

```
@@ -249,7 +249,7 @@ public function afterException($controller, $methodName, \Exception $exception):
 					$url = $this->urlGenerator->linkToRoute('core.login.showLoginForm', $params);
 					$response = new RedirectResponse($url);
 				} else {
-					$response = new TemplateResponse('core', '403', ['file' => $exception->getMessage()], 'guest');
+					$response = new TemplateResponse('core', '403', ['message' => $exception->getMessage()], 'guest');
```

maybe trying to assign predefined parameters should throw an error
And only work via a ->forceAssign method?